docker/dockercli.go

```diff
@@ -59,7 +61,22 @@ func (cli *dockerCLI) startContainer(ctx context.Context, containerName string)
 	defer ioReader.Close()
 
 	ui.Note().Msg("creating docker container")
-	resp, err := cli.dockerClient.ContainerCreate(ctx, cli.cfg.containerConfig, cli.cfg.containerHostConfig, nil, nil, containerName)
+
+	endpoints := make(map[string]*network.EndpointSettings, 1)
+	endpoints[cli.cfg.networkName] = &network.EndpointSettings{}
+
+	networkConfig := &network.NetworkingConfig{
+		EndpointsConfig: endpoints,
+	}
+
+	resp, err := cli.dockerClient.ContainerCreate(
+		ctx,
+		cli.cfg.containerConfig,
+		cli.cfg.containerHostConfig,
+		networkConfig,
```

Comment:
Maybe we can use default network if no network name is specified by the user.

Reply:
Talked with @florindragos in slack and implemented same functionality as before if no network is specified.